src/modules/home/api.ts

```diff
@@ -1,16 +1,18 @@
 import axios from "axios";
 
+const BASE_URL = process.env.REACT_APP_BASE_URL;
+
 export async function getPopularRestaurantsFromAPI(){
-    const result = await axios.get('http://localhost:5000/restaurants/popular');
+    const result = await axios.get(`${BASE_URL}/restaurants/popular`);
```

Comment:
instead of using different paths for popular, chef of the week and signature dishes you can utilize query parameters